### ISSUE TITLE
Error make libzauth

### ISSUE THREAD

**Issue by wireappru** [wireapp/wire-server#365]
When you try to compile a package, libzauth issues the following error. OS is used by Ubuntu 16.04 x64

```
~/wire-server/libs/libzauth# make
make -C libzauth-c dist
make[1]: Entering directory '/root/wire-server/libs/libzauth/libzauth-c'
cargo build --release
   Compiling libsodium-sys v0.1.0
error: failed to run custom build command for `libsodium-sys v0.1.0`
process didn't exit successfully: `/root/wire-server/libs/libzauth/libzauth-c/target/release/build/libsodium-sys-c1bafbb032bf1833/build-script-build` (exit code: 101)
--- stdout
cargo:rerun-if-env-changed=SODIUM_LIB_DIR
cargo:rerun-if-env-changed=SODIUM_INC_DIR
cargo:rerun-if-env-changed=SODIUM_STATIC
cargo:rustc-link-search=native=/usr/lib/x86_64-linux-gnu
cargo:rustc-link-lib=sodium

--- stderr
thread 'main' panicked at 'Unable to find libclang: "couldn\'t find any of [\'libclang.so\', \'libclang.so.*\', \'libclang-*.so\'], set the LIBCLANG_PATH environment variable to a path where one of these files can be found (skipped: [])"', libcore/result.rs:945:5
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::print
             at libstd/sys_common/backtrace.rs:71
             at libstd/sys_common/backtrace.rs:59
   2: std::panicking::default_hook::{{closure}}
             at libstd/panicking.rs:207
   3: std::panicking::default_hook
             at libstd/panicking.rs:223
   4: std::panicking::rust_panic_with_hook
             at libstd/panicking.rs:402
   5: std::panicking::begin_panic_fmt
             at libstd/panicking.rs:349
   6: rust_begin_unwind
             at libstd/panicking.rs:325
   7: core::panicking::panic_fmt
             at libcore/panicking.rs:72
   8: core::result::unwrap_failed
   9: std::sync::once::Once::call_once::{{closure}}
  10: std::sync::once::Once::call_inner
             at libstd/sync/once.rs:340
  11: bindgen::Builder::generate
  12: build_script_build::main
  13: std::rt::lang_start::{{closure}}
  14: std::panicking::try::do_call
             at libstd/rt.rs:59
             at libstd/panicking.rs:306
  15: __rust_maybe_catch_panic
             at libpanic_unwind/lib.rs:102
  16: std::rt::lang_start_internal
             at libstd/panicking.rs:285
             at libstd/panic.rs:361
             at libstd/rt.rs:58
  17: main
  18: __libc_start_main
  19: _start

Makefile:27: recipe for target 'build-release' failed
make[1]: *** [build-release] Error 101
make[1]: Leaving directory '/root/wire-server/libs/libzauth/libzauth-c'
Makefile:2: recipe for target 'dist' failed
make: *** [dist] Error 2

```

**Comment by neongreen** [comment]
Known; see https://github.com/wireapp/wire-server/pull/364.

**Comment by jschaul** [comment]
Fixed by #364
